### PR TITLE
feat: migration

### PR DESCRIPTION
**Pull request by GabrielPrediger**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -132,7 +132,7 @@
     "@babel/plugin-transform-runtime": "^7.21.4",
     "@babel/preset-env": "^7.21.5",
     "@nomiclabs/hardhat-ethers": "^2.2.3",
-    "@types/node": "^18.16.3",
+    "@types/node": "^20.14.11",
```
Comment (lucaslovato) [inline package.json:135]: go back to 18v

File: source/scripts/migrations/faucet.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import { reload } from 'utils/browser';
```
Comment (lucaslovato) [inline source/scripts/migrations/faucet.ts:1]: remove unused import

File: source/scripts/migrations/faucet.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    } else {
+      newState.faucetModal = {
+        ...newState.faucetModal,
+        57: true,
+        570: true,
+        5700: true,
+        57000: true,
+      };
+    }
+    console.log(newState, `pos`);
```
Comment (lucaslovato) [inline source/scripts/migrations/faucet.ts:24]: remove this log

File: package.json (hunk 1 above)
Comment (GabrielPrediger) [reply package.json:135]: fixed

File: source/scripts/migrations/faucet.ts (hunk 3 above)
Comment (GabrielPrediger) [reply source/scripts/migrations/faucet.ts:24]: removed

**Review by lucaslovato — APPROVED** [review]

**Pull request merged**